src/config.rs

```diff
@@ -920,6 +923,34 @@ impl Config {
     }
 }
 
+fn parse_shell(raw: &String) -> Option<Vec<CString>> {
```

Comment:
Sure, so you want an array in the config file instead of a single string? Do you want the user to set the first argument as well?